### DISCUSSION TITLE
How to set up an oral suspension for administration

### DISCUSSION THREAD

**Discussion by redamancy-1** [Open-Systems-Pharmacology/Forum#1143] [Questions & Problems]
How to set up an oral suspension for administration
Well-meaning people help~
![image](https://user-images.githubusercontent.com/100193905/155094871-4009009f-9bd0-44e0-a509-1456a1a92155.png)

**Comment by Yuri05** [comment]
Please check 
https://docs.open-systems-pharmacology.org/working-with-pk-sim/pk-sim-documentation/pk-sim-formulations
and
https://docs.open-systems-pharmacology.org/working-with-pk-sim/pk-sim-documentation/pk-sim-simulations#select-administration

You might also check some video tutorials:
https://github.com/Open-Systems-Pharmacology/OSP-based-publications-and-content/issues/199
https://github.com/Open-Systems-Pharmacology/OSP-based-publications-and-content/issues/202
